src/backend/downloadmanager/downloadqueue.ts

```diff
@@ -133,7 +133,11 @@ async function addToQueue(element: DMQueueElement) {
   } else {
     const installInfo = await libraryManagerMap[
       element.params.runner
-    ].getInstallInfo(element.params.appName, element.params.platformToInstall)
+    ].getInstallInfo(
+      element.params.appName,
+      element.params.platformToInstall,
+      element.params.channelName ?? 'main'
```

Comment:
do we still want the `?? 'main'` here? I don't see how this is different than `channelNameToInstall = 'main'` in the hp library

Reply:
Not really, because in this method from `downloadqueue` it never comes as `undefined` since we always pass the channel to the `install` command and this ` ?? ` was just to satisfy typescript.

But not always when calling `getInstallInfo` it gets the channel and sometimes it gets `undefined`. 
I think it might be from a call somewhere else or a race condition where it miss the channel the first time and the second goes fine.

Reply:
are any error messages logged when it comes back `undefined`? @flavioislima

Reply:
Only that release meta is undefined as a result. Since it gets nothing from an undefined channel.
That's is why the return.